ODF-community.french.md

```diff
@@ -0,0 +1,526 @@
+<tr valign="top">
+<td rowspan=3 width=33%><strong>Solliciter usagers + collaborateurs</strong><br/>
+ • Rassembler des retours auprès d'usagers internes et de personnes qui vont
+vous aider (questionnaires, entretiens, groupes témoins, etc.)<br/><br/>
+ • Faciliter la participation + les gestion de projet. Demander aux usagers
+quels outils de collaboration ils préfèrent utiliser. Prévoir la consolidation
+et le publication du feedback.<br/><br/>
+ • Rester ouvert à de nouvelles informations et perspectives<Br/><br/>
+ • Envisager le feedback et la commununication pair-à-pair, en plus des canaux
+formels. <br/><br/>
+
+<strong>Définir les attentes dès le départ</strong><Br/>
+ • Etre spécifique à propos des types de feedback que vous recherchez et à
+propos de qui pris les décisions. <Br/><Br/>
+ • Publier le processus de décision et la planification du projet plan avec des
+roles, des dates et des contraintes
+</td>
+<td rowspan=3 width=33%><strong>Expliquer les évidences</strong><Br/>
+ • Publier le périmètre du projet ou de la décision et itérer régulièrement<br/><br/>
+ • Publier les facteurs de décisions et leur importance relative<Br/><Br/>
+ • Publier vos recherches, notamment les compromis difficiles, les pré-requis de
+l'entreprise<br/><br/>
+ • Dans le mesure du possible, publier tous les éléments légaux, les rapports et
+les enjeux de confidentialité<br/><br/>
+
+<strong>Planifier la transition</strong><br/><br/>
+ • Developper et rassembler le feedback sur la communication, la gestion du
+changement et les plans d'adoption<br/><Br/>
+ • Imaginez comment des personnes ennervées pourraient répondre ( via un memo
+ ou un autre canal)
+</td>
+<td>
+ • Comment allons-nous prendre les décisions?<Br/><Br/>
+ • Quel usagers internes, parties prenantes et collaborateurs allons-nous impliquer ?<br/><br/>
+ • Comment allons-nous les solliciter et communiquer avec eux ?<br/><br/>
+ • Quels sont les options open source ?<br/><br/>
+ • Comment le choix d'une technologie ou d'un format propriétaire va limiter
+nosc choix dans le futur ?<br/><br/>
+ • Comment cela s'aligne avec la stratégie et la mission de la société ?<br/><br/>
+ • Où est-ce que cela peut coincer avec les valeurs et la culture de Red Hat ?
+</td>
+</tr>
+<tr>
+<td><strong>Points Clés</strong></td>
+</tr>
+<tr>
+<td>
+ • Impact – qui, quelle fréquence et les imprévus<br/>
+ • Où et comment collaborer<Br/>
+ • Roles + responsibilités (modèle OPT : https://github.com/red-hat-people-team/opt-model/) 
+</td>
+</tr>
+</table>
+
+<br/>
+
+Phase ou activité: Concevoir, Developper, Tester <a name="design-develop-test"></a>
+----------------------------
+
+_page 9 sur 15_ 
+
+<table>
+<tr>
+<td colspan=2 width="66%"><strong>Étapes pour devenir ouvert</strong></td>
+<td><strong>Questions à poser</strong></td>
+</tr>
+<tr valign="top">
+<td rowspan=3 width=33%><strong>Batir votre communauté</strong><br/>
+ • Demander aux départments qui parmi leur équipe peut fournir du feedback<Br/><Br/>
```

Comment:
Alos, there is usually more than one team, so I think we should use plural for "leur équipe".